src/banman.cpp

```diff
@@ -67,14 +67,36 @@ void BanMan::ClearBanned()
     if (m_client_interface) m_client_interface->BannedListChanged();
 }
 
+int BanMan::IsBannedLevel(CNetAddr net_addr)
+{
+    // Returns the most severe level of banning that applies to this address.
+    // 0 - Not banned
+    // 1 - Automatic misbehavior ban
+    // 2 - Any other ban
```

Comment:
Nit: maybe swap `1` and `2` so we can add other / more detailed specific reasons later?

Reply:
meh.   Trivial to do if there is a need. Without a need otherwise the current order makes more sense to me.

Reply:
This data is ephemeral, right? Not saved into `banlist.dat` (which would make it more tedious to change later).